telco-core/configuration/core-finish.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: core-customer-policies
+policyDefaults:
+  namespace: ztp-core-policies
+  policySets: []
+  placement:
+    clusterSelectors:
+      common: "core"
+      finish-version-4.18: ""
+  remediationAction: "inform"
+policies:
+  # unpause baseline configuration
+  - name: custom-mcp-unpause
+    policyAnnotations:
+      ran.openshift.io/ztp-deploy-wave: "200"
+    manifests:
+      # The MCPs should be added as extra manifests at installation time. Their
+      # inclusion here will reset the pause to false, allowing the MCPs to update
+      - path: ../install/custom-manifests/mcp-worker-1.yaml
+        patches:
+        - spec:
+            paused: false
+          status:
+            conditions:
+            - type: Updated
+              status: "True"
+            - type: Updating
+              status: "False" 
```

Comment:
```suggestion
              status: "False"
```
From the lint check there is an extra space at the end of this line.

Reply:
fixed